R/deployDoc.R

```diff
@@ -53,6 +53,11 @@ standardizeSingleDocDeployment <- function(path,
     taskComplete(quiet, "Document dependencies discovered")
 
     appFiles <- c(basename(path), resources$path)
+
+    if (file.exists(file.path(dirname(path), ".Rprofile"))) {
```

Comment:
What about `.Renviron`?

Reply:
I can see an argument both ways for `.Renviron`...

Reply:
I think I give the most weight to the fact that `.Renviron` usually contains secrets so they shouldn't be uploaded to a file that other people could access. Plus my goal is to provide more tools for env vars in https://github.com/rstudio/rsconnect/issues/667.